include/RAJA/pattern/teams/teams_core.hpp

```diff
@@ -132,12 +132,13 @@ struct Resources {
   Teams teams;
   Threads threads;
   Lanes lanes;
+  const char *kernel_name;
```

Comment:
I'm not sure if the default ctor is ever used but it would be good to always init this to `nullptr`:

```
const char *kernel_name = nullptr;
```